quill-core/src/main/scala/io/getquill/dsl/QueryDsl.scala

```diff
@@ -13,6 +13,13 @@ private[dsl] trait QueryDsl {
   @compileTimeOnly(NonQuotedException.message)
   def querySchema[T](entity: String, columns: (T => (Any, String))*): EntityQuery[T] = NonQuotedException()
 
+  implicit class NullableColumnExtensions[A](o: Option[A]) {
```

Comment:
if we keep these methods, they should be `@compileTimeOnly` and throw `NonQuotedException()`

Reply:
Cool. Will do.

Reply:
They're all removed